sample/kotlinapp/src/main/java/com/marcinmoskala/kotlinapp/notification/NotificationPublisher.kt

```diff
@@ -16,18 +19,36 @@ class NotificationPublisher : BroadcastReceiver() {
 
     override fun onReceive(context: Context, intent: Intent) {
         ActivityStarter.fill(this, intent)
+
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O)
+        {
+            val channel = NotificationChannel(NOTIFICATION_CHANNEL_ID, "activitystarter", NotificationManager.IMPORTANCE_HIGH)
+            context.notificationManager.createNotificationChannel(channel)
+        }
+
         val notification = createSimpleNotification(context)
         context.notificationManager.notify(id, notification)
     }
 
-    fun createSimpleNotification(context: Context) = Notification.Builder(context)
+    fun createSimpleNotification(context: Context): Notification
+    {
+        val builder = Notification.Builder(context)
             .setContentTitle(getTextTitle(id))
             .setContentText(getTextSubtitle(time))
             .setSmallIcon(R.mipmap.ic_launcher)
-            .build()!!
+
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O)
+        {
```

Comment:
It doesn't look like Kotlin style
https://kotlinlang.org/docs/reference/coding-conventions.html